srcpkgs/vscode/template

```diff
@@ -46,7 +46,10 @@ do_build() {
 	/usr/lib/node_modules/npm/bin/node-gyp-bin/node-gyp install \
 	--target=$_electronver \
 	--tarball=/usr/include/electron${_electronver%%.*}/node_headers.tar.gz
-	yarn install --ignore-engines --arch=x64
+
+	npm install -g npm@latest && \
+		yarn install --ignore-engines --arch=x64
```

Comment:
```suggestion
	npm install -g npm@latest
	yarn install --ignore-engines --arch=x64
```
Why do we need to update npm?

Reply:
Otherwise built failed ... Guess because node-gyp uses it own outdated version of npm.